ReleaseNotes.md

```diff
@@ -1,11 +1,42 @@
 # Release Notes
 
-## Introduction
-*Neural Network Compression Framework (NNCF)* is a toolset for Neural Networks model compression.
-The framework organized as a Python module that can be built and used as standalone or within
-samples distributed with the code.  The samples demonstrate the usage of compression methods on
-public models and datasets for three different use cases: Image Classification, Object Detection,
-and Semantic Segmentation.
+## New in Release 2.4.0
+Target version updates:
+- Bump target framework versions to PyTorch 1.13.1, TensorFlow 2.8.x, ONNX opset 13
+- Increased target HuggingFace transformers version for the integration patch to 4.23.1
+
+Features:
```

Comment:
JPQD for several models has been added. I think it should be stated either as an experimental/preliminary support or for particular model.

Reply:
Done